docs/md/meta.md

```diff
@@ -74,25 +74,25 @@ entt::meta_factory<my_type> factory{};
 The returned value is a _factory object_ to use to continue building the meta
 type.
 
-By default, a meta type is associated with the identifier returned by the
+By default, a meta-type is associated with the identifier returned by the
```

Comment:
To be honest, I prefer that `meta-type`, `meta-data` and all the other `meta-` stuff stays the way they were before, i.e. without the `-`. This form is more common in general, and it also avoids confusion with things like `meta-default constructor`, which feels wrong.

Reply:
default `meta-constructor`?

Reply:
Fair enough. 👍